x-pack/plugin/runtime-fields/qa/core-with-search/src/yamlRestTest/java/org/elasticsearch/xpack/runtimefields/test/search/CoreTestsWithSearchRuntimeFieldsIT.java

```diff
@@ -116,20 +111,22 @@ protected boolean modifySearch(ApiCallSection search) {
                         return mergeMappings(new String[] { "*" });
                     }
                     String[] patterns = Arrays.stream(index.split(",")).map(m -> m.equals("_all") ? "*" : m).toArray(String[]::new);
+                    // TODO this is always false?
```

Comment:
@nik9000 this is for you I think :)

Reply:
ooops. I think I mean `length == 1`. I can take a look in a follow up.